packages/lit/src/lib/Theme/index.ts

```diff
@@ -0,0 +1,69 @@
+import { ThemeSchema } from '@nl-design-system-community/design-tokens-schema';
+import startTokens from '@nl-design-system-unstable/start-design-tokens/dist/tokens.json';
+import StyleDictionary from 'style-dictionary';
+import { DesignTokens } from 'style-dictionary/types';
+
+export default class Theme {
+  static readonly defaults = ThemeSchema.parse(startTokens); // Start tokens are default for all Themes
+  #defaults: DesignTokens; // Every Theme has private defaults to revert to.
+  #tokens: DesignTokens = {}; // In practice this will be set via the this.tokens() setter in the constructor
+  #stylesheet: CSSStyleSheet = new CSSStyleSheet();
+
+  constructor(tokens?: DesignTokens) {
+    // @TODO: make sure that parsed tokens conform to DesignTokens type;
+    this.#defaults = structuredClone(tokens || (Theme.defaults as DesignTokens));
+    this.tokens = structuredClone(this.#defaults);
+  }
+
+  get defaults() {
+    return this.#defaults;
+  }
+
+  get tokens() {
+    return this.#tokens;
+  }
+
+  set tokens(values: DesignTokens) {
+    this.#tokens = values;
+    this.toCSS().then((css) => {
+      const sheet = this.#stylesheet;
+      sheet.replace(css);
+    });
+  }
+
+  get stylesheet() {
+    return this.#stylesheet;
+  }
```

Comment:
I'm in doubt whether I should expose the asynchronous nature of the stylesheet updates, and whether that should be done via this property. I'll leave it for now because it is a clearer interface this way, and the stylesheet should be passed only once for use as an adopted stylesheet